sources/packages/web/src/assets/css/formio-shared.scss

```diff
@@ -31,6 +32,21 @@ $border-color-info: #6554c0;
   color: $primary-header-color !important;
 }
 
+.secondary-btn-background {
+  background: $secondary-header-color !important;
+  color: $button-font-color !important;
+}
```

Comment:
i don't think we need to add a custom class, when vuetify is released we can create a v-btn  component, as I showed you yesterday, it has props color and text color, that can directly be passed